ci/glci/model.py

```diff
@@ -272,6 +272,14 @@ class GcpPublishedImage(PublishedImageBase):
     gcp_project_name: str
 
 
+@dataclasses.dataclass(frozen=True)
+class AzurePublishedImage:
+    transport_state: str
+    urn: str
+    publish_operation_id: str
```

Comment:
do we actually need to persist the publish_operation_id?

Reply:
Hmm I think so. It is an output of the first step (upload image. + trigger publish process). In the second step we need this id to poll the status of the publishing process. Same for the `golive_operation_id`